stack_definition.json

```diff
@@ -454,6 +456,15 @@
 				{
 					"name": "member_host_flavor",
 					"value": "b3c.4x16.encrypted"
+				},
+				{
+					"name": "service_credential_names",
+					"value": {
+						"elasticsearch_admin" : "Administrator",
+						"elasticsearch_reader" : "Operator",
+						"elasticsearch_viewer" : "Viewer",
+						"elasticsearch_editor" : "Editor"
+					}
```

Comment:
Is this the approach that Watson team want? Or do they want to use admin_password?

Reply:
We probably don't want to be creating 4 credentials here anyway

Reply:
I talked with Watson team, and they decided to go with 
```
"value": {
						"elastic_db_admin" : "Administrator",
						"wxasst_db_user" : "Editor",
						"toolchain_db_user" : "Editor"
					}
```					
they prefer IAM credentials over user/pass